{
    "checksum": "0a8cf92cefdcb8ff76d09854649f1c17ff191b9dc48b853bf01ac3fe9d187120",
    "file": "errors.py",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/x-python"
    },
    "number_of_duplicates": 3,
    "package": "alot",
    "path": "alot/0.9.1-2/alot/db/errors.py",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/alot/0.9.1-2/",
        "metric": {
            "size": 1728
        },
        "pts_link": "https://tracker.debian.org/pkg/alot",
        "sloc": [
            [
                "python",
                11107
            ],
            [
                "makefile",
                132
            ]
        ],
        "suites": [
            "bookworm",
            "bullseye",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/python-team/applications/alot",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/a/alot/0.9.1-2/alot/db/errors.py",
    "stat": {
        "perms": "rw-r--r--",
        "size": 539,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "0.9.1-2"
}